requirements.txt

```diff
@@ -21,7 +21,7 @@ xmltodict == 0.12.0
 oauth2client == 4.1.3
 pika == 0.13.0
 git+https://github.com/metabrainz/brainzutils-python.git@v1.18.1
-spotipy == 2.16.1
+spotipy == 2.17.1
```

Comment:
I see https://github.com/plamere/spotipy/releases `2.18.0` is out. Let's upgrade directly to that.

Reply:
Also, relook at https://github.com/metabrainz/listenbrainz-server/pull/1361#discussion_r630037327